usaspending_api/database_scripts/etl/award_delta_view.sql

```diff
@@ -66,47 +48,41 @@ SELECT
   vw_es_award_search.funding_toptier_agency_code,
   vw_es_award_search.awarding_subtier_agency_code,
   vw_es_award_search.funding_subtier_agency_code,
-  CASE
-    WHEN vw_es_award_search.funding_toptier_agency_name IS NOT NULL
-      THEN CONCAT(
-        '{"name":"', vw_es_award_search.funding_toptier_agency_name,
-        '","code":"', vw_es_award_search.funding_toptier_agency_code,
-        '","id":"', (SELECT a1.id FROM agency a1 WHERE a1.toptier_agency_id = (SELECT a2.toptier_agency_id FROM agency a2 WHERE a2.id = vw_es_award_search.funding_agency_id) ORDER BY a1.toptier_flag DESC, a1.id LIMIT 1), '"}'
-      )
-    ELSE NULL
-  END AS funding_toptier_agency_agg_key,
-  CASE
-    WHEN vw_es_award_search.funding_subtier_agency_name IS NOT NULL
-      THEN CONCAT(
-        '{"name":"', vw_es_award_search.funding_subtier_agency_name,
-        '","code":"', vw_es_award_search.funding_subtier_agency_code,
-        '","id":"', (SELECT a1.id FROM agency a1 WHERE a1.toptier_agency_id = (SELECT a2.toptier_agency_id FROM agency a2 WHERE a2.id = vw_es_award_search.funding_agency_id) ORDER BY a1.toptier_flag DESC, a1.id LIMIT 1), '"}'
-      )
-    ELSE NULL
-  END AS funding_subtier_agency_agg_key,
+  vw_es_award_search.funding_toptier_agency_agg_key,
+  vw_es_award_search.funding_subtier_agency_agg_key,
 
   vw_es_award_search.recipient_location_country_code,
   vw_es_award_search.recipient_location_country_name,
   vw_es_award_search.recipient_location_state_code,
+  vw_es_award_search.recipient_location_state_name,
```

Comment:
Appears that a number of new fields are being added. Are they necessary for Elasticsearch?

Reply:
Removed `recipient_location_state_name`, `recipient_location_state_fips`, `recipient_location_state_population`, `recipient_location_county_population`, `recipient_location_congressional_population`, `pop_state_name`, `pop_state_fips`, `pop_state_population`, `pop_county_population`, and `pop_congressional_population`